==== triton/lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp#12 - /google/src/cloud/akuegel/fix_triton/triton/lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp ====
# action=edit type=text
--- triton/lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp	2023-07-07 06:59:50.000000000 -0700
+++ triton/lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp	2023-08-03 02:59:01.000000000 -0700
@@ -533,7 +533,7 @@
     // newShape = rank_reduce(shape)
     // Triton only supports static tensor sizes
     SmallVector<Value, 4> strideVals;
-    for (auto i = 0; i < op.static_sizes().size(); ++i) {
+    for (auto i = 0; i < op.getStaticSizes().size(); ++i) {
       if (op.getStaticSize(i) == 1) {
         offsetVals.erase(offsetVals.begin() + i);
       } else {
==== triton/lib/Dialect/TritonGPU/IR/Dialect.cpp#20 - /google/src/cloud/akuegel/fix_triton/triton/lib/Dialect/TritonGPU/IR/Dialect.cpp ====
# action=edit type=text
--- triton/lib/Dialect/TritonGPU/IR/Dialect.cpp	2023-07-12 07:02:26.000000000 -0700
+++ triton/lib/Dialect/TritonGPU/IR/Dialect.cpp	2023-08-03 03:00:20.000000000 -0700
@@ -1201,10 +1201,10 @@
     auto newArg = rewriter.create<triton::gpu::ConvertLayoutOp>(
         op->getLoc(), newType, extract_slice.getSource());
     rewriter.replaceOpWithNewOp<triton::gpu::ExtractSliceOp>(
-        op, resType, newArg.getResult(), extract_slice.offsets(),
-        extract_slice.sizes(), extract_slice.strides(),
-        extract_slice.static_offsets(), extract_slice.static_sizes(),
-        extract_slice.static_strides());
+        op, resType, newArg.getResult(), extract_slice.getOffsets(),
+        extract_slice.getSizes(), extract_slice.getStrides(),
+        extract_slice.getStaticOffsets(), extract_slice.getStaticSizes(),
+        extract_slice.getStaticStrides());
     return mlir::success();
   }
 
